.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,24 @@
+name: Rubocop
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_ONLY: rubocop
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Ruby 2.7.5
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.5
```

Comment:
Why not leave `ruby-version` unspecified so that setup-ruby simply uses `.ruby-version`?

Reply:
> Why not leave `ruby-version` unspecified so that setup-ruby simply uses `.ruby-version`?

because I didn't realise that was an option 😅 happy to change that